Modelica/Blocks/Sources.mo

```diff
@@ -593,12 +605,18 @@ and that the parameter <code>startTime</code> is omitted since the voltage can b
     parameter Real amplitude=1 "Amplitude of sine wave"
     annotation(Dialog(groupImage="modelica://Modelica/Resources/Images/Blocks/Sources/Sinc.png"));
     parameter SI.Frequency f(start=1) "Frequency of sine wave";
+    parameter Boolean continuous = false "If we want a smooth signal, the output will start at offset + amplitude";
```

Comment:
Similar to above:
```suggestion
    parameter Boolean continuous = false "Make output continuous by starting at offset + amplitude rather than just offset";
```

Reply:
In the terminology I'm used to, smooth is the same as $C^{\infinity}$, so a function that is just one times continuously differentiable is not smooth.  This is in agreement with the definition on Wikipedia (see _bump function_):
- https://en.wikipedia.org/wiki/Smoothness#Differentiability_classes

Reply:
There are different terminologies - at the start of that page "At the very minimum, a function could be considered smooth if it is differentiable everywhere (hence continuous)." That's why the operator in Modelica is called smooth, and why people discuss smooth interpolation using splines - https://en.wikipedia.org/wiki/Spline_(mathematics)

So it doesn't make smooth wrong, just ambigous.

Reply:
True, but here we can avoid some confusion by use less ambiguous terminology.  (That the operator is called `smooth` doesn't bother me, as I think of it as just a nice and more compact name for an operator, compared to something more descriptive such as `smoothness` or even `orderOfSmoothness`.)

Reply:
Looks good now.  Resolving.